### PR TITLE
Full To Do App

### PR DESCRIPTION
**Pull request by NicolasAndrehh**
# Milestone - Project Completed
This pull request includes the following changes:
- Setup React Application
- Setup linters
- Add header component 
- Add taskInput component 
- Add taskRow component 
- Add toDoList component 
- Add function to create a new task
- Add function to delete a task
- Add function to edit a task description
- Add webpage styling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,197 @@
...
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Math Magicians React <a name="about-project"></a>
```
Comment (blueberry1312) [inline README.md:34]: You forgot to change this part of the past project

**Comment by blueberry1312** [comment]
## STATUS: APPROVED 🎉🎉🎉

Hello @NicolasAndrehh !

Great job setting up the react todo list app!. Now you can merge it!

git-merge


## Highlights 

Correct linters ✅
Great use of gitflow ✅
Good code design and implementation ✅

## Required Changes ♻️

None


### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

N/A

Cheers and Happy coding!👏

File: README.md (hunk 1 above)
Comment (blueberry1312) [inline README.md]: [OPTIONAL]
_Remember to update all sections in your Readme._

File: public/manifest.json
Comment (blueberry1312) [inline public/manifest.json:1]: [OPTIONAL]
_Remember to remove all files that CRA templates create and youre not using_

File: public/logo512.png
Comment (blueberry1312) [inline public/logo512.png:1]: [OPTIONAL]
_Remember to remove all files that CRA templates create and youre not using_

File: public/logo192.png
Comment (blueberry1312) [inline public/logo192.png:1]: [OPTIONAL]
_Remember to remove all files that CRA templates create and youre not using_

**Comment by blueberry1312** [comment]
[OPTIONAL]
**_For better UX, I recommend to you to add a button to clear all completed tasks!!_**

File: README.md [hunk 2]
```diff
@@ -1,2 +1,197 @@
...
+- **[This project use React]**
+- **[This project does a correct use of GitHub Flows]**
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[To Do React - Live Demo](https://nicolasandrehh.github.io/To-Do-React/)
```
Comment (blueberry1312) [inline README.md:81]: **The live Demo is not working, remember to fix using: [Part 11 – Deploying React app to GitHub Pages](https://ibaslogic.com/deploying-react-app-to-github-pages/)**

**Review by blueberry1312 — APPROVED** [review]

**Comment by NicolasAndrehh** [comment]
Hey @blueberry1312!
Thank you so much for your feedback, I'll take into count all your suggestions for an upcoming update of the application 😄👍

**Pull request merged**